test/test_qna.py

```diff
@@ -29,13 +28,14 @@ async def test(testing_bot, guild_id):
 async def test_question(testing_bot):
     print('testing question')
     qna_channel = discord.utils.get(testing_bot.get_all_channels(), name='q-and-a')
-    await qna_channel.send('!ask \"Hello\"')
-
-    sleep(0.5)
-
-    messages = await qna_channel.history(limit=1).flatten()
+    await qna_channel.send('!ask Hello')
 
+    sleep(.5)
```

Comment:
What is exactly the purpose of using sleep if we are using async and await?

Reply:
To be honest, I'm not entirely sure, it's just the way the last group had it coded but I changed it a couple of times to see which is why it counted as a change on my end.